Cargo.lock

```diff
@@ -194,9 +363,9 @@ dependencies = [
  "bitflags 1.3.2",
  "bytes",
  "futures-util",
- "http",
- "http-body",
- "hyper",
+ "http 0.2.12",
+ "http-body 0.4.6",
+ "hyper 0.14.28",
```

Comment:
The upcoming `tonic 0.12` migrates to `http 1`, so once that is released almost all of these particular duplicate dependencies will disappear (except for one crate that Arti depends on at the version we're running).